frontend/integration-tests/tests/cluster-settings.scenario.ts

```diff
@@ -0,0 +1,220 @@
+    await clusterSettingsView.downloadButton.click();
+    await crudView.isLoaded();
+
+    expect(clusterSettingsView.viewSchemaButton.isDisplayed()).toBe(true);
+    await clusterSettingsView.viewSchemaButton.click();
+    await crudView.isLoaded();
+
+    expect(clusterSettingsView.viewDetailsButton.get(0).isDisplayed()).toBe(true);
+    await clusterSettingsView.viewDetailsButton.get(0).click();
+    await crudView.isLoaded();
+  });
+  it('display Global Configuration page, click Configuration Resource item link and display details', async() => {
+
+    await horizontalnavView.clickHorizontalTab('Global Configuration');
+    await crudView.isLoaded();
+
+    expect(clusterSettingsView.linkForGlobalConfig.get(4).isPresent()).toBe(true);
+    await clusterSettingsView.linkForGlobalConfig.get(4).click();
+    await crudView.isLoaded();
+
+
+    await horizontalnavView.clickHorizontalTab('YAML');
+    await crudView.isLoaded();
+
+    const content = await yamlView.getEditorContent();
+    const newContent = _.defaultsDeep({}, {metadata: {labels}, spec: {text}}, safeLoad(content));
+    await yamlView.setEditorContent(safeDump(newContent));
+    await yamlView.isLoaded();
+
+    expect(clusterSettingsView.saveChangesButton.isDisplayed()).toBe(true);
+    await clusterSettingsView.saveChangesButton.click();
+    await crudView.isLoaded();
+
+    expect(clusterSettingsView.reloadObjectButton.isDisplayed()).toBe(true);
+    await clusterSettingsView.reloadObjectButton.click();
+    await crudView.isLoaded();
+
+    expect(yamlView.getEditorContent()).toContain(`app: ${testName}-app`);
+    expect(yamlView.getEditorContent()).toContain(`text: ${name} Logs`);
+
+    expect(clusterSettingsView.downloadButton.isDisplayed()).toBe(true);
+    await clusterSettingsView.downloadButton.click();
+    await crudView.isLoaded();
+
+    expect(clusterSettingsView.viewSchemaButton.isDisplayed()).toBe(true);
+    await clusterSettingsView.viewSchemaButton.click();
+    await crudView.isLoaded();
+
+    expect(clusterSettingsView.viewDetailsButton.get(0).isDisplayed()).toBe(true);
+    await clusterSettingsView.viewDetailsButton.get(0).click();
+    await crudView.isLoaded();
+  });
+  it('display Global Configuration page, click dropdown link to edit resource and display details', async() => {
+
+    await horizontalnavView.clickHorizontalTab('Global Configuration');
+    await crudView.isLoaded();
+
+    expect(clusterSettingsView.globalConfigKebabButton.get(0).isDisplayed()).toBe(true);
+    await clusterSettingsView.globalConfigKebabButton.get(0).click();
+    await crudView.isLoaded();
+
+    expect(clusterSettingsView.globalConfigKebabDropDownButton.get(0).isDisplayed()).toBe(true);
+    await clusterSettingsView.globalConfigKebabDropDownButton.get(0).click();
+    await crudView.isLoaded();
+
+    const content = await yamlView.getEditorContent();
+    const newContent = _.defaultsDeep({}, {metadata: {labels}, spec: {text}}, safeLoad(content));
```

Comment:
Again I'd skip modifying anything here. Just make sure the YAML editor displays.